gulpTasks/dev.js

```diff
@@ -7,70 +7,69 @@ const buildUtilities = require('../gulpTasks/buildUtilities.js');
 const _ = require('underscore');
 const path = require('path');
 const fs = require('fs');
+const glob = require('glob');
 
-let webpackConfig = require('../webpack.config.js');
+const webpackConfig = require('../webpack.config.js');
 webpackConfig.entry['CoveoJsSearch.Lazy'].unshift('webpack-dev-server/client?http://localhost:8080/');
 const compiler = webpack(webpackConfig);
 
-let webpackConfigTest = require('../webpack.test.config.js');
+const webpackConfigTest = require('../webpack.test.config.js');
 webpackConfigTest.entry['tests'].unshift('webpack-dev-server/client?http://localhost:8081/');
 const compilerTest = webpack(webpackConfigTest);
 
-let debouncedLinkToExternal = _.debounce(()=> {
+let server;
```

Comment:
Since `server` is now in a higher scope, we can define what it is here instead of inside gulp.task('dev') and use `const`.

Reply:
It's just that if I define it in the global scope of the scripts, it gets created every time we start any gulp tasks (since all scripts are parsed by gulp).

And it's a relatively long function to execute. So by putting it inside the 'dev' task, it's only creating the server when we actually need it.

Reply:
Okay, makes sense.